website/content/en/docs/kubebuilder/advanced-topics.md

```diff
@@ -60,7 +60,7 @@ import (
   routev1 "github.com/openshift/api/route/v1"
 )
 
-func main() {
+func init() {
```

Comment:
These inits are scaffolded like:
```go
func init() {
        _ = clientgoscheme.AddToScheme(scheme)

        _ = cachev1alpha1.AddToScheme(scheme)
        // +kubebuilder:scaffold:scheme
}
```
Would be good to align with this syntax, even if errors should be checked here.

Reply:
The add actually are as `utilruntime.Must(kbatchv1.AddToScheme(scheme))` since https://github.com/kubernetes-sigs/kubebuilder/pull/1462.  I mean with `utilruntime.Must(<add>)`